app/controllers/users_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  include ControllerHelper
```

Comment:
use `...Helper`s for views only
if there are some shared functionality, have a look at the concerns

Reply:
done